e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/validation/multiple-contract-policy-validation-maximum-buyer-will-owe.spec.js

```diff
@@ -146,4 +155,17 @@ context('Insurance - Policy and exports - Multiple contract policy page - form v
       );
     });
   });
+
+  describe('when maximum buyer will owe is valid and contains a comma', () => {
+    it('should redirect to the next page as all fields are valid', () => {
+      cy.completeAndSubmitMultipleContractPolicyForm();
+      partials.backLink().click();
+
+      multipleContractPolicyPage[MAXIMUM_BUYER_WILL_OWE].input().clear().type('1,234');
```

Comment:
Can we please add a case with `1,234.123`.

Reply:
Good shout, i've added these in for all the monetary fields in policy and exports. Note, comma and decimal will cause validation error (we enforce no decimals, whole numbers only).